napari/_qt/resources/build_icons.py

```diff
@@ -10,9 +10,11 @@
 from subprocess import SubprocessError, check_call
 from typing import Dict, List, Tuple
 
-from ..utils.theme import _themes
+import napari.resources
```

Comment:
well the "reason" was that i felt like it was clearer to put `import napari.resources` than to say `from ... import resources`... (particularly since `from .. import resources` is _also_ a real import ...)  
but i get that it's not strictly in keeping with the relative imports we use.  so let me know if you want it changed

Reply:
lol, we can bump this one to @jni. I don't have strong feelings, was just thinking about consistency, but I might say that having both `from ... import resources` and `from .. import resources` be valid suggests we might want to rename one of them! We could take a cue from how we deal with utils and go with `qt_resources` for the inner one. (In which case this would still be `from ... import resources`). We don't always do that with utils, but might even want to too.

I'll let @jni weigh in though too, he might have a better idea!

Reply:
renamed `_qt.resources` to `_qt.qt_resources`